tensorflow/core/ops/cudnn_rnn_ops.cc

```diff
@@ -135,7 +135,52 @@ REGISTER_OP("CudnnRNNV2")
       int dir_count = (direction == "bidirectional") ? 2 : 1;
       DimensionHandle output_size;
       TF_RETURN_IF_ERROR(c->Multiply(num_units, dir_count, &output_size));
-      auto output_shape = c->MakeShape({seq_length, batch_size, output_size});
+      auto output_shape = c->MakeShape({max_seq_length, batch_size, output_size});
+      auto output_h_shape = input_h_shape;
+      auto output_c_shape TF_ATTRIBUTE_UNUSED =
+          (rnn_mode == "lstm") ? output_h_shape : c->MakeShape({});
+      c->set_output(0, output_shape);
```

Comment:
Why do we change V2?

Reply:
What does this mean?

Reply:
This changes CudnnV2, what's the reason?

Reply:
I think I only changes seq_length -> max_seq_length for CudnnRNNV2. I didn't change anything else for CudnnRNNV2. Do you mean something else?

Reply:
I guess when you compare the changes, github misaligns v2 and v3 (which is just below it).

Reply:
I see your points. I changed back.